clippy.toml

```diff
@@ -0,0 +1,2 @@
+# Don't warn about these identifiers when using clippy::doc_markdown.
+doc-valid-idents = ["RenderDoc", ".."]
```

Comment:
clippy toml is no longer needed this can go into Cargo.toml now under `[lints.clippy]`

Reply:
I don't think this part can .. I did move enabling the warnings into there though.

Reply:
didn't find a way to do this on cargo.toml either :/
I made `RenderDoc` mention in the docs now a link, working around this 🤷 😁